packages/components/src/components/Scrollbar/index.tsx

```diff
@@ -1,40 +0,0 @@
-import { SCREEN_SIZE } from '../../config/variables';
-import { css } from 'styled-components';
-
-// For nice scrollbars in Chrome put these styles in the global stylesheet.
-// For Firefox you need to apply `scrollbar-color` prop on per element basis
-const scrollbarStyles = css`
-    /* don't apply on mobile devices as they use very usable auto-hide scrollbars out of the box */
-
-    @media (min-width: ${SCREEN_SIZE.SM}) {
-        ::-webkit-scrollbar {
-            background-color: transparent;
-            width: 10px;
-        }
-
-        /* background of the scrollbar except button or resizer */
-        ::-webkit-scrollbar-track {
-            background-color: transparent;
-        }
-
-        /* scrollbar itself */
-        ::-webkit-scrollbar-thumb {
-            /* 7F7F7F for mac-like color */
-            background-color: ${props => props.theme.SCROLLBAR_THUMB};
```

Comment:
you could also delete `SCROLLBAR_THUMB` color from config

Reply:
8bcea11f64e81337f5f4c3ad870f4c0f203890b9